bin/rebuild

```diff
@@ -39,46 +38,23 @@ main() {
         git clone https://github.com/nextstrain/ncov.git
     fi
 
-    if [[ -n "${metadata_src:-}" && -n "${sequences_src:-}" ]]; then
-        # Run on these exact data files by uploading them with the build context for
-        # the Batch job, instead of roundtripping them through S3.
-        #
-        # Since the Batch job is submitted before the rest of the GitHub workflow
-        # updates the files on S3, this also avoids a narrow race condition if AWS
-        # Batch is quick or the upload to S3 slow.
-        mkdir -p ncov/data/
-        cp -v "$metadata_src" "$sequences_src" ncov/data/
-    fi
-
-
     local output aws_batch_job_id
 
     output=$(
         "$bin"/launch-build \
-            ncov deploy upload \
+            ncov \
+            --profile ${profile_name} \
+            deploy upload \
             --config slack_token="$SLACK_TOKEN"
     )
 
     echo "$output"
 
     # Extract the AWS job ID from the `nextstrain build --aws-batch --detach` output
-    aws_batch_job_id=$(grep "AWS Batch Job ID" <<<"$output" | cut -d ' ' -f 5)
-
-    echo "Notifying Slack about rebuild."
-    "$bin"/notify-slack "A new build was submitted. Follow along in your local ncov repo with: "'```'"nextstrain build --aws-batch --attach $aws_batch_job_id ."'```'
-}
-
-# Returns 1 if both files match (have identical hashes). Else returns 0.
-hashes-match() {
-    local dst_s3path="${2#s3://}"
-    local dst_bucket="${dst_s3path%%/*}"
-    local dst_key="${dst_s3path#*/}"
-
-    local src_hash dst_hash no_hash=0000000000000000000000000000000000000000000000000000000000000000
-    src_hash="$("$bin/sha256sum" < "$1")"
-    dst_hash="$(aws s3api head-object --bucket "$dst_bucket" --key "$dst_key" --query Metadata.sha256sum --output text || echo "$no_hash")"
+    aws_batch_job_id=$(grep "AWS Batch Job ID" <<<"$output" | cut -d ' ' -f 5) || 1
```

Comment:
What does ` || 1` do here?

Reply:
Thanks for noticing this! I must have added this unintentionally & have removed it.

Reply:
Can never tell with bash code if it is some clever syntax I don't recognize!